src/DataValues/TimeValue.php

```diff
@@ -79,27 +87,31 @@ class TimeValue extends DataValueObject {
 	/**
 	 * @since 0.1
 	 *
-	 * @param string $time an ISO 8601 date and time
+	 * @param string $time timestamp in a format depending on the calendar model
 	 * @param int $timezone offset from UTC in minutes
```

Comment:
We should have a way to say "timezone is undefined".
We may actually drop this field, and use the time zone encoded in the timestamp, since time zones are not really defined for historical calendars anyway.